examples/http/auth-and-tls/main.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"context"
+	"log"
+	"time"
+
+	qdb "github.com/questdb/go-questdb-client/v3/pkg/http"
```

Comment:
nit: do we need the `/pkg` section in the package path? Why not `github.com/questdb/go-questdb-client/v3/http`?

Reply:
No reason, just trying to follow go conventions. I can move `http` and `tcp` out of `pkg/`

Reply:
> No reason, just trying to follow go conventions.

Do you refer to https://go.dev/blog/package-names (couldn't find /pkg convention there) or it's something else?

Reply:
I was using the golang standards repo (https://github.com/golang-standards/project-layout/tree/master/pkg) where `pkg` is used for library code that can be imported by external projects

Reply:
I see. That's not official convention. Interestingly, it simply follows an old convention from Golang source code:
> The pkg directory origins: The old Go source code used to use pkg for its packages and then various Go projects in the community started copying the pattern

Reply:
Most go projects I've seen use `pkg` these days. Interesting to learn that it's not an official convention